vendor/clink.lua

```diff
@@ -361,27 +365,29 @@ end
 
 ---
 -- Get the status of working dir
--- @return {bool}
+-- @return {bool|'branchonly'}
```

Comment:
Making the return type polymorphic adds extra burden on call sites, and adds potential for regression in unexpected situations.